### ISSUE TITLE
Is there a way to detach a spconv tensor?

### ISSUE THREAD

**Issue by aldipiroli** [traveller59/spconv#491]
Is there an equivalent of `.detach()` from pytorch?
